### PR TITLE
[pota-quantization-value-test] Revise test

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit revises the test to reduce the test duration.

### Before
```
Start 58: pota_fake_wquant_test
58/76 Test #58: pota_fake_wquant_test ..................   Passed    4.87 sec
      Start 59: pota_record_minmax_test
59/76 Test #59: pota_record_minmax_test ................   Passed    9.06 sec
      Start 60: pota_quantization_test
60/76 Test #60: pota_quantization_test .................   Passed    4.95 sec
      Start 61: pota_fake_wquant_test_with_config
61/76 Test #61: pota_fake_wquant_test_with_config ......   Passed    3.87 sec
      Start 62: pota_parallel_record_minmax_test
62/76 Test #62: pota_parallel_record_minmax_test .......   Passed    9.66 sec
      Start 63: pota_quantization_test_with_config
63/76 Test #63: pota_quantization_test_with_config .....   Passed    7.94 sec
      Start 64: pota_wo_quantization_test
64/76 Test #64: pota_wo_quantization_test ..............   Passed    0.61 sec
```

### After
```
Test project /home/seongwoo/ONE/build
    Start 59: pota_fake_wquant_test
1/7 Test #59: pota_fake_wquant_test ................   Passed    0.51 sec
    Start 60: pota_record_minmax_test
2/7 Test #60: pota_record_minmax_test ..............   Passed    0.77 sec
    Start 61: pota_quantization_test
3/7 Test #61: pota_quantization_test ...............   Passed    0.74 sec
    Start 62: pota_fake_wquant_test_with_config
4/7 Test #62: pota_fake_wquant_test_with_config ....   Passed    0.48 sec
    Start 63: pota_parallel_record_minmax_test
5/7 Test #63: pota_parallel_record_minmax_test .....   Passed    0.83 sec
    Start 64: pota_quantization_test_with_config
6/7 Test #64: pota_quantization_test_with_config ...   Passed    1.06 sec
    Start 65: pota_wo_quantization_test
7/7 Test #65: pota_wo_quantization_test ............   Passed    0.14 sec

100% tests passed, 0 tests failed out of 7

Total Test time (real) =   4.54 sec
```

Related: #12071 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
After landing this PR, the test time will be alleviated.

```bash
$ ./nncc test -j8

100% tests passed, 0 tests failed out of 76

Total Test time (real) =   7.56 sec
```

**Comment by mhs4670go** [comment]
For reviewers, this PR introduces python scripts that do what they do all at once. I mean that the script is running on a single recipe or test now but the introducing script is gonna run all the recipe at once. It is for reducing the latency that happens when the python script is called.

File: compiler/pota-quantization-value-test/test_fake_wquant.sh [hunk 1]
```diff
@@ -55,12 +57,6 @@ while [ "$1" != "" ]; do
...
 
-    # Compare result
-    "${VIRTUALENV}/bin/python" "${COMPARE_SCRIPT_PATH}" \
-      --input_h5 "${TEST_RESULT_FILE}.fake_quantized.circle.h5" \
-      --expect_dir "${SOURCE_PATH}/expected_outputs/${MODELNAME}/${GRANULARITY}/${DTYPE}/fake_quantization" \
-      --mode fake_quantization
-
     if [[ $? -eq 0 ]]; then
       touch "${PASSED_TAG}"
     fi
```
Comment (jinevening) [inline compiler/pota-quantization-value-test/test_fake_wquant.sh:62]: I guess that passed file is generated regardless of the result of comparison?

Comment (mhs4670go) [reply compiler/pota-quantization-value-test/test_fake_wquant.sh:62]: @jinevening I'll remove the flow of pass file creation. Instead, I just print the related logs at the end of the comparing script. Below is an example for the log.

```bash
Start 61: pota_quantization_test
3/7 Test #61: pota_quantization_test ...............***Failed    0.60 sec
-- Found CIRCLE_QUANTIZER: /home/seongwoo/ONE/build/compiler/circle-quantizer/circle-quantizer
-- Found CIRCLE_TENSORDUMP: /home/seongwoo/ONE/build/compiler/circle-tensordump/circle-tensordump
-- Found workdir: /home/seongwoo/ONE/build/compiler/common-artifacts
~/ONE/build/compiler/common-artifacts ~/ONE/build/compiler/pota-quantization-value-test
~/ONE/build/compiler/pota-quantization-value-test
FAILED
- Add_002.channel.int16
ERROR: Quantized scale of ifm1 ([0.00014653]) do not match with expected value (110.00014653272228).

- AveragePool2D_000.channel.int16
ERROR: Quantized zero_point of ifm ([0]) do not match with expected value (110.0).
```

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**